docs/source/gbd2017_models/causes/neoplasms/cancers/cancer_model.rst

```diff
@@ -0,0 +1,131 @@
+.. _2017_cancer_model_cancers:
+
+=======
+Cancers
+=======
+
+This is a common document for the cancers that are modelled similarly in GBD. This document currently has information related to the following cancers:
```

Comment:
does this mean that they're also going to have the same model structure within vivarium?

Reply:
yes